prompt/prompt-executor/prompt-executor-clients/prompt-executor-bedrock-client/build.gradle.kts

```diff
@@ -15,6 +15,7 @@ kotlin {
                 api(project(":agents:agents-tools"))
                 api(project(":agents:agents-utils"))
                 api(project(":prompt:prompt-executor:prompt-executor-clients"))
+                implementation(project(":prompt:prompt-executor:prompt-executor-clients:prompt-executor-anthropic-client"))
```

Comment:
Why did you add it to `commonMain`? It should be not needed here since we provide only JVM implementation for Bedrock, and this dependency is already included in the `jvmMain` block.